department-of-veterans-affairs/caseflow
# Add positive feedback to Attorney Evaluation Screen for AMA appeals

Connects #7929

### Description
Allow judge to leave positive feedback for attorney in addition to negative

### Acceptance Criteria
- [ ] Attorney feels better about their work upon receiving positive feedback

### Testing Plan
1. Choose BVAAABSHIRE (Judge with hearings & cases)
2. Go to Queue
3. Choose an AMA case
4. Select "Ready for Dispatch"
5. Fill out decision modal
6. Verify that "Identify Positive Feedback" section is functional

### Documentation Updates
- [ ] Topline File Descriptions Added or Updated as Needed

*Schema Changes Only*
